app/views/shared/_polls.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="p_1 filled_block content_block" style="background-image: url(<%= image_url("illustrations/emojis.svg") %>)">
+  <h2><%= t("shared.polls.opinion") %></h2>
+  <% if next_poll.is_a?(::GobiertoParticipation::Poll) %>
```

Comment:
`next_poll(poll_id)`, right?

Reply:
In that case to avoid calling next_poll two times assign it to a variable `poll = next_poll(poll_id)` and check if poll is present.